docs/Displaying a User Interface/Adaptive Interface Capabilities/index.md

```diff
@@ -135,11 +148,17 @@ sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.APP_S
 ```
 !@
 
+@!
+```js
+const appServicesCapabilities = sdlManager.getSystemCapabilityManager().getCapability(SDL.rpc.enums.SystemCapabilityType.APP_SERVICES);
+```
```

Comment:
The other examples are using the asynchronous definition for getting capabilities, which means that `updateCapability` should be used instead of `getCapability`, and it can be awaited on:

`const appServicesCapabilities = await sdlManager.getSystemCapabilityManager().updateCapability(SDL.rpc.enums.SystemCapabilityType.APP_SERVICES);`